pkg/event/event.go

```diff
@@ -221,6 +222,121 @@ func (e *event) Tenant() tenant.Id {
 	return e.tid
 }
 
+func (e *event) evalArrayPath(path string, curr interface{}) (interface{}, interface{}, string, int) {
```

Comment:
Can we write some unit test that specifically test this function? I know you have some table test above that already test this, but I think a more specific test directly on the function will allow us to test more cases, and also, bump up our code coverage ;)

Reply:
done